pwiz_tools/Skyline/Properties/Resources.zh-CHS.resx

```diff
@@ -3995,11 +3995,11 @@
     <value>..\Resources\koina-logo.fdf731d5.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
   <data name="CommandLine_ImportPeptideList_Importing_peptide_list__0__from_file__1____" xml:space="preserve">
-    <value>正在导入文件 {1} 中的肽段列表 {0}…</value>
+    <value>正在导入文件 {1} 中的肽段列表 {0}...</value>

```

Comment:
Chinese to Roman punctuation change

Reply:
Knowing enough Chinese to embarrass myself talking to children, I was curious to learn about the Chinese version of the ellipsis. Apparently it's formally six dots, usually two of those 3-dot characters in a row. https://www.zdic.net/ts/fulu/2019/05/27.html
But if the translators were just using one of the 3-dot characters, obviously go with that. :P It's more consistent with the English version that way.